providers/ms365/resources/ms365.lr

```diff
@@ -397,6 +399,26 @@ private microsoft.security.securityscore @defaults("id azureTenantId") {
   vendorInformation dict
 }
 
+// Microsoft Entra users who are at risk
+microsoft.security.riskyUser @defaults("principalName riskLevel riskState lastUpdatedAt"){
+  // Risky user ID
+  id string
+  // User name
+  name string
+  // User principal
+  principalName string
```

Comment:
nit: maybe we can use the `microsoft.security.user` resource here ?

Reply:
Yeah, lets remove it for now and stick with user reference. Great idea.

Reply:
Ok. I reviewed the code. I think we should keep `principalName` for now. Otherwise we always need to create all user objects. I would like to avoid that right now.